Cargo.toml

```diff
@@ -33,11 +33,12 @@ dpdk-sysroot-helper = { path = "./dpdk-sysroot-helper" }
 dplane-rpc = { git = "https://github.com/githedgehog/dplane-rpc.git", version = "1.0.0" }
 errno = { path = "./errno", package = "dataplane-errno" }
 id = { path = "./id", package = "dataplane-id" }
-net = { path = "./net" }
+net = { path = "./net", features = ["test_buffer"]}
```

Comment:
I don't know if this one is required if you use the feature in `dataplane/Cargo.toml` already, although I already asked you about something similar in a previous PR and you said it was necessary so maybe it is here as well. Just give a quick try to see if it's removable, if you haven't already.